examples/fluids/index.md

```diff
@@ -185,21 +190,75 @@ Our formulation follows {cite}`hughesetal2010`, which offers a comprehensive rev
   \int_{\Omega} \bm v \cdot \left( \frac{\partial \bm{q}_N}{\partial t} - \bm{S}(\bm{q}_N) \right)  \,dV
   - \int_{\Omega} \nabla \bm v \!:\! \bm{F}(\bm{q}_N)\,dV & \\
   + \int_{\partial \Omega} \bm v \cdot \bm{F}(\bm{q}_N) \cdot \widehat{\bm{n}} \,dS & \\
-  + \int_{\Omega} \bm{P}(\bm v)^T \, \nabla \cdot \bm{F} \, (\bm{q}_N) \,dV
+  + \int_{\Omega} \mathcal{P}(\bm v)^T \, \nabla \cdot \bm{F} \, (\bm{q}_N) \,dV
   & = 0 \, , \; \forall \bm v \in \mathcal{V}_p
   \end{aligned}
   $$ (eq-weak-vector-ns-su)
 
   This stabilization technique can be selected using the option `-stab su`.
 
-In both {eq}`eq-weak-vector-ns-su` and {eq}`eq-weak-vector-ns-supg`, $\bm{P} \,$ is called the *perturbation to the test-function space*, since it modifies the original Galerkin method into *SUPG* or *SU* schemes.
+In both {eq}`eq-weak-vector-ns-su` and {eq}`eq-weak-vector-ns-supg`, $\mathcal P$ is called the *perturbation to the test-function space*, since it modifies the original Galerkin method into *SUPG* or *SU* schemes.
 It is defined as
 
 $$
-\bm{P}(\bm v) \equiv \left(\bm{\tau} \cdot \frac{\partial \bm{F} \, (\bm{q}_N)}{\partial \bm{q}_N} \right)^T \, \nabla \bm v\,,
+\mathcal P(\bm v) \equiv \left(\bm{\tau} \cdot \frac{\partial \bm{F}_{\text{adv}} (\bm{q}_N)}{\partial \bm{q}_N} \right)^T \, \nabla \bm v\,,
+$$
+
+where parameter $\bm{\tau} \in \mathbb R^{3\times 3}$ is an intrinsic time scale matrix.
+This expression contains the flux Jacobian, which we express in variational notation by differentiating the advective flux $\bm F_{\text{adv}}$ of {eq}`eq-ns-flux`
+
+$$
+\begin{aligned}
+\diff\bm F_{\text{adv}}(\diff\bm q; \bm q) &= \frac{\partial \bm F_{\text{adv}}}{\partial \bm q} \diff\bm q \\
```

Comment:
@LeilaGhaffari The idea here is that `\frac{\partial \bm F_{\text{adv}}}{\partial \bm q}` is the (5,3,5) tensor that we don't actually form because we really only need its action. IMO, this makes notation and implementation for differentiation more mechanical, though it doesn't create the full tensor to be fed to an eigensolver or otherwise inspected. The function `dF(dq; q)` can be used to build the matrix by running it on `dq` chosen as columns of the identity.